Repairs/app/src/main/java/com/rooio/repairs/NavigationBar.kt

```diff
@@ -13,39 +13,58 @@ import androidx.transition.TransitionManager
 
 abstract class NavigationBar : RestApi() {
 
+    private lateinit var navBar: ViewGroup
+    private lateinit var collapseText: TextView
+    private lateinit var dashboardText: TextView
+    private lateinit var jobsText: TextView
+    private lateinit var  equipmentText: TextView
+    private lateinit var  settingsText: TextView
+    private lateinit var collapse: ImageView
+    private lateinit var divider: View
+    private lateinit var dashboardImage: ImageView
+    private lateinit var settingsImage: ImageView
+    private lateinit var equipmentImage: ImageView
+    private lateinit var jobsImage: ImageView
+    private lateinit var coloredDashboard: ImageView
+    private lateinit var coloredSettings: ImageView
+    private lateinit var coloredJobs: ImageView
+    private lateinit var coloredEquipment: ImageView
+    private lateinit var transitionsContainer: ViewGroup
+    private lateinit var transitionsContainer2: ViewGroup
+
+    companion object {
+        var visible = false
```

Comment:
I think naming this variable "collapsed" instead of "visible" might be a more suitable name?

Reply:
Good idea!